### PR TITLE
feat: add preset layout grid with nesting 

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#493
Closes SAP/fundamental-styles#297

## Description
Adding preset grids for users to use up to 16 columns and compliance to fiori 3 specs. In progress for IE

fd-layout-grid--gap-1 class removed

### Before:
Nothing before

### After:
Screen Shot 2020-01-10 at 11 38 59 AM

### CODE REVIEW

**Comment by mikerodonnell89** [comment]
Needs visual tests updated

File: src/layout-grid.scss [hunk 1]
```diff
@@ -8,12 +8,28 @@
...
-  grid-gap: var(--fd-width-gutter, $fd-width--gutter);
+  grid-template-columns: 100%;
+  grid-gap: 1rem;
+
+  @media only screen and (max-width: 599px) {
+    grid-gap: 0.5rem;
+  }
+
+  * {
+    height: fit-content;
```
Comment (salarenko) [inline src/layout-grid.scss]: We shouldn't use `fit-content` in our styles. It is [not supported](https://caniuse.com/#search=fit-content) in many browsers

**Comment by JKMarkowski** [comment]
@stefanoScalzo The main problem there is ofc IE11, which doesn't entirely support [grid](https://caniuse.com/#search=grid). Maybe let's follow the bootstrap 4. There is grid system based on flex. It should work on ie11.

**Comment by stefanoScalzo** [comment]
Goign to fixed in a pr that does not use css grids

**Pull request closed without merging**